packages/twenty-server/src/workspace/workspace-query-builder/factories/update-one-query.factory.ts

```diff
@@ -31,12 +31,17 @@ export class UpdateOneQueryFactory {
       options.fieldMetadataCollection,
     );
 
+    const argsData = {
+      ...computedArgs.data,
+      updatedAt: new Date().toISOString(),
+    };
+
```

Comment:
Let me know if this is okay or it is supposed to be wrapped with a typed definition

Reply:
I think we are happy with this for now!